app/dal/jpa/JPAJobOfferRepository.java

```diff
@@ -138,22 +138,29 @@ private JobOffer update(EntityManager em, JobOffer offer) {
         return jpaApi.withTransaction(function);
     }
 
-    private String count(EntityManager em) {
-        Query q = em.createQuery("SELECT COUNT (j) FROM JobOffer j");
+    private String count(EntityManager em, String companies) {
+        Query q;
+        if (companies != null && !companies.isEmpty()) {
+            List<String> companyList = Arrays.asList(companies.split(","));
+            q = em.createQuery("SELECT COUNT (j) FROM JobOffer j WHERE company_uuid IN :companies")
+            .setParameter("companies",companyList);
+        } else{
+            q = em.createQuery("SELECT COUNT (j) FROM JobOffer j");
```

Comment:
```suggestion
            q = em.createQuery("SELECT COUNT (j) FROM JobOffer j WHERE j.open = true");
```

Maak deze nog even zodat hij alleen de count van alle open joboffers pakt.